content/tutorials/manage-the-rippled-server/installation/build-run-rippled-in-reporting-mode.md

```diff
@@ -0,0 +1,362 @@
+        brew services start postgres
+
+2. Connect to the PostgreSQL Database Server using psql and create a user `newuser` and a database `reporting`.
+
+        psql postgres
+	        CREATE ROLE newuser WITH LOGIN PASSWORD ‘password’;
+            ALTER ROLE newuser CREATEDB;
+        \q
+        psql postgres -U newuser
+        postgres=# create database reporting;
+
+#### Install and Configure the Primary Persistent Datastore 
+
+**Cassandra** 
+
+Install Cassandra and then create a keyspace for `rippled`, with replication. 
+
+While a replication factor of 3 is recommended, when running locally, replication is not needed and you can set `replication_factor` to 1.    
+        
+    $ cqlsh [host] [port]
+    > CREATE KEYSPACE `rippled` WITH REPLICATION =
+    {'class' : 'SimpleStrategy', 'replication_factor' : 1    };
+
+**NuDB** 
+
+If you’re running `rippled` in reporting mode for your local network, you can choose to use NuDB instead of Cassandra as your backend database. 
+
+NuDB is installed as part of your `rippled` build setup and does not require any additional installation steps.
+
+
+### Procedure
+
+1. Build `rippled` for reporting mode on [Ubuntu](build-run-rippled-ubuntu.html) or [macOS](build-run-rippled-macos.html).
+
+    <!-- MULTICODE_BLOCK_START -->
+
+    *Linux*
+
+        wget https://github.com/Kitware/CMake/releases/download/v3.13.3/cmake-3.13.3-Linux-x86_64.sh
+    
+        sudo sh cmake-3.13.3-Linux-x86_64.sh --prefix=/usr/local --exclude-subdir -Dreporting=ON -DCMAKE_BUILD_TYPE=Debug ..
+
+
+    *macOS*
+
+        cmake -G "Unix Makefiles" -Dreporting=ON -DCMAKE_BUILD_TYPE=Debug ..
+
+        cmake --build . -- -j 4
+
+    <!-- MULTICODE_BLOCK_END -->
+
+2. Create a configuration file to run `rippled` in reporting mode. 
+
+    Make a copy of the example config file, `rippled.cfg`, and save it as `reporting-mode.cfg` in a location that enables you to run `rippled` as a non-root user. For example:
+    
+        cd /etc/opt/ripple/
+
+        mkdir -p $HOME/.config/ripple
+
+        cp cfg/rippled-example.cfg $HOME/.config/ripple/rippled-reporting-mode.cfg
+
+3. Edit rippled-reporting-mode.cfg to set necessary file paths. The user you plan to run `rippled` as must have write permissions to all of the paths you specify here.
+
+    1. Set the `[node_db]` path to the location where you want to store the ledger database.
+
+    2. Set the `[database_path]` to the location where you want to store other database data. (This includes an SQLite database with configuration data, and is typically one level above the `[node_db]` path field.)
+        
+    3. Set the `[debug_logfile]` to a path where `rippled` can write logging information.
+
+    Note that these are the only configurations required for `rippled` to start up successfully. All other configurations are optional and can be tweaked after you have a working server.
+
+4. Edit the `rippled-reporting-mode.cfg` file to enable Reporting Mode: 
```

Comment:
We should make sure that the case of "reporting mode" is treated consistently. I think it's not a proper noun so we should put it in lower case (except as appropriate for headings, starts of sentences, etc.).

```suggestion
4. Edit the `rippled-reporting-mode.cfg` file to enable reporting mode: 
```